src/main/java/Storage.java

```diff
@@ -0,0 +1,72 @@
+import java.io.File;
```

Comment:
This class seems to stand alone from the main program. You can link it to the save method in the main file